src/components/Note/Note.tsx

```diff
@@ -58,7 +60,7 @@ const Note = (props: NoteProps) => {
   drag(noteRef);
   drop(noteRef);
 
-  const filteredVotes = filterVotes(props.votes, props.activeVoting);
+  const filteredVotes = filterVotes(props.votes, props.activeVoting, props.voteConfiguration?.showVotesOfOtherUsers);
```

Comment:
Maybe you could already filter the votes in the `routes/Board/Board.tsx` component when they are retrieved from the store

Reply:
Yes. Done.